adapters/adgeneration/adgeneration_test.go

```diff
@@ -174,3 +175,82 @@ func TestCreateAd(t *testing.T) {
 		t.Errorf("%v does not match createAd.", adgVastResponse)
 	}
 }
+
+func TestMakeBids(t *testing.T) {
+	bidder := NewAdgenerationAdapter("https://d.socdm.com/adsv/v1")
+	internalRequest := &openrtb.BidRequest{
+		ID: "test-success-bid-request",
+		Imp: []openrtb.Imp{
+			{ID: "bidRequest-success-test", Banner: &openrtb.Banner{Format: []openrtb.Format{{W: 300, H: 250}}}, Ext: json.RawMessage(`{"bidder": { "id": "58278" }}`)},
+		},
+		Device: &openrtb.Device{UA: "testUA", IP: "testIP"},
+		Site:   &openrtb.Site{Page: "https://supership.com"},
+		User:   &openrtb.User{BuyerUID: "buyerID"},
+	}
+	externalRequest := adapters.RequestData{}
+	response := adapters.ResponseData{
+		StatusCode: 200,
+		Body:       ([]byte)("{\n \"ad\": \"<!DOCTYPE html>\\n<head>\\n<meta charset=\\\"UTF-8\\\">\\n<script src=\\\"test.com\\\"></script>\\n<body>\\n<div id=\\\"medibasspContainer\\\">\\n<iframe src=\\\"https://dummy-iframe.com></iframe>\\n</div>\\n</body>\\n\",\n \"beacon\": \"<img src=\\\"https://dummy-beacon.com\\\">\",\n \"beaconurl\": \"https://tg.socdm.com/bc/v3?b=beaconID&t=.gif\",\n \"cpm\": 30,\n \"creativeid\": \"Dummy_supership.jp\",\n \"displaytype\": \"1\",\n \"h\": 250,\n \"ids\": {\n \"anid\": \"\",\n \"diid\": \"\",\n \"idfa\": \"\",\n \"soc\": \"Soc\"\n },\n \"location_params\": null,\n \"locationid\": \"58278\",\n \"results\": [\n {\n \"ad\": \"<div id=\\\"medibasspContainer\\\">\\n<iframe src=\\\"https://dummy-iframe.com></iframe>\\n</div>\",\n \"beacon\": \"<img src=\\\"https://dummy-beacon.com\\\">\",\n \"beaconurl\": \"ttps://dummy-beacon.com\",\n \"cpm\": 30,\n \"creativeid\": \"Dummy_supership.jp\",\n \"h\": 250,\n \"landing_url\": \"\",\n \"scheduleid\": \"512601\",\n \"trackers\": {\n \"imp\": [\n \"https://tg.socdm.com/bc/v3?b=beaconID.gif\"\n ],\n \"viewable_imp\": [\n \"https://tg.socdm.com/aux/inview\"\n ],\n \"viewable_measured\": [\n \"https://tg.socdm.com/aux/measured\"\n ]\n },\n \"ttl\": 10,\n \"w\": 300,\n \"weight\": 1\n }\n ],\n \"dealid\": \"test-deal-id\",\n \"rotation\": \"0\",\n \"scheduleid\": \"512601\",\n \"sdktype\": \"0\",\n \"trackers\": {\n \"imp\": [\n \"https://tg.socdm.com/bc/v3?b=beaconID.gif\"\n ],\n \"viewable_imp\": [\n \"https://tg.socdm.com/aux/inview\"\n ],\n \"viewable_measured\": [\n \"https://tg.socdm.com/aux/measured\"\n ]\n },\n \"ttl\": 10,\n \"w\": 300\n}"),
```

Comment:
For the purposes of this unit test, could you please use a simplified  request? For example, the "ad" could just me "testAd", and the "adM" could be "testAdM". This works, but is difficult to read. You can also use the json files to represent these two tests cases which is designed for improved readability as well.